tensorflow/python/kernel_tests/array_ops/manip_ops_test.py

```diff
@@ -25,12 +25,9 @@
 from tensorflow.python.platform import test as test_lib
 
 # pylint: disable=g-import-not-at-top
-try:
-  from distutils.version import StrictVersion as Version
-  # numpy.roll for multiple shifts was introduced in numpy version 1.12.0
-  NP_ROLL_CAN_MULTISHIFT = Version(np.version.version) >= Version("1.12.0")
-except ImportError:
-  NP_ROLL_CAN_MULTISHIFT = False
+from packaging.version import StrictVersion as Version
+# numpy.roll for multiple shifts was introduced in numpy version 1.12.0
```

Comment:
Can you remove these lines please and move the import immediately after the numpy import?

Reply:
Sorry, keep the numpy.roll comment, but delete line 31 (import not-at-top)